### PR TITLE
Feature/submit application popover

### PR DESCRIPTION
**Pull request by Ajiva-D**

### CODE REVIEW

File: packages/grants/components/Modal/DualActionModal.tsx [hunk 1]
```diff
@@ -1,20 +1,21 @@
-import { CircularProgress } from "@material-ui/core";
-import React from "react";
+import Button from "components/CommonComponents/Button";
+// import MainActionButton from "components/MainActionButton";
+// import SecondaryActionButton from "components/SecondaryActionButton";
```
Comment (arne-z) [inline packages/grants/components/Modal/DualActionModal.tsx:3]: in or out?

File: packages/grants/components/Modal/DualActionModal.tsx [hunk 2]
```diff
@@ -52,39 +72,30 @@ export const DualActionModal: React.FC<DualActionModalProps> = ({
...
-                type="button"
-                disabled={progress}
-                className="mt-3 w-full inline-flex justify-center rounded-md border border-gray-300 shadow-sm px-4 py-2 bg-white text-base font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 sm:mt-0 sm:col-start-1 sm:text-sm"
-                onClick={() => onDismiss.onClick && onDismiss.onClick()}
-              >
+              </Button>
+            )}
+            {onDismiss && (
+              <Button variant="secondary" onClick={() => dismiss()} className="py-2 px-5 w-full">
+                {" "}
```
Comment (arne-z) [inline packages/grants/components/Modal/DualActionModal.tsx:94]:
```suggestion
```

File: packages/grants/components/Modal/DualActionModal.tsx [hunk 3]
```diff
@@ -52,39 +72,30 @@ export const DualActionModal: React.FC<DualActionModalProps> = ({
...
-              <button
-                type="button"
-                disabled={progress}
-                className="w-full inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-indigo-600 text-base font-medium text-white hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 sm:col-start-2 sm:text-sm"
-                onClick={() => onConfirm.onClick && onConfirm.onClick()}
-              >
+          <div className="mt-5 sm:mt-6 sm:grid sm:grid-cols-2 sm:gap-3 sm:grid-flow-row-dense">
+            {onConfirm && (
+              <Button variant="primary" onClick={() => confirm()} className="py-2 px-5 w-full">
+                {" "}
```
Comment (arne-z) [inline packages/grants/components/Modal/DualActionModal.tsx:88]:
```suggestion
```

File: packages/grants/components/Modal/DualActionWideModal.tsx [hunk 4]
```diff
@@ -73,40 +93,44 @@ const Example: React.FC<DualActionWideModalProps> = ({ content, title, visible,
...
-              <div className="mt-5 sm:mt-6 sm:grid sm:grid-cols-2 sm:gap-3 sm:grid-flow-row-dense">
+              <div className="mt-5 sm:mt-6 sm:grid sm:grid-cols-2 sm:gap-4 sm:grid-flow-row-dense">
                 {onConfirm && (
-                  <button
-                    type="button"
-                    className="w-full inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-indigo-600 text-base font-medium text-white hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 sm:col-start-2 sm:text-sm"
-                    onClick={() => confirm()}
-                  >
+                  <Button variant="primary" onClick={() => confirm()} className="py-3 px-5">
+                    {" "}
```
Comment (arne-z) [inline packages/grants/components/Modal/DualActionWideModal.tsx:125]:
```suggestion
```

File: packages/grants/components/Modal/DualActionWideModal.tsx [hunk 5]
```diff
@@ -73,40 +93,44 @@ const Example: React.FC<DualActionWideModalProps> = ({ content, title, visible,
...
-
                 {onDismiss && (
-                  <button
-                    type="button"
-                    className="mt-3 w-full inline-flex justify-center rounded-md border border-gray-300 shadow-sm px-4 py-2 bg-white text-base font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 sm:mt-0 sm:col-start-1 sm:text-sm"
-                    ref={cancelButtonRef}
-                    onClick={() => dismiss()}
-                  >
+                  <Button variant="secondary" onClick={() => dismiss()} className="py-3 px-5">
+                    {" "}
```
Comment (arne-z) [inline packages/grants/components/Modal/DualActionWideModal.tsx:131]:
```suggestion
```

File: packages/grants/components/Modal/SingleActionModal.tsx [hunk 6]
```diff
@@ -1,86 +1,160 @@
...
+                        {title}
+                      </h3>
+                      <div className="py-6">
+                        {children ? children : <p className="text-base md:text-sm text-gray-500">{content}</p>}
+                      </div>
+                    </div>
+                  </div>
+                  <div className="">
+                    <Button variant="primary" onClick={confirm} className="py-2 px-5 w-full">
+                      {" "}
```
Comment (arne-z) [inline packages/grants/components/Modal/SingleActionModal.tsx:147]:
```suggestion
```

**Review by arne-z — APPROVED** [review]

File: packages/grants/components/Modal/DualActionModal.tsx (hunk 1 above)
Comment (Ajiva-D) [reply packages/grants/components/Modal/DualActionModal.tsx:3]: should be out

**Pull request merged**